src/templates/patterns.js

```diff
@@ -107,6 +116,15 @@ export const aboutPageQuery = graphql`
             }
           }
         }
+        featuredimage: patternArt {
+          childImageSharp {
+            resize(width: 1200) {
```

Comment:
I'll have to ask graphics/social media for guidance on an image meant to be 1200/1200. Alternatively - we could try the resize param of `fit: 'contain'` but I prefer the first honestly.